dgraph/sample/schema.gql

```diff
@@ -0,0 +1,19 @@
+type Entity {
+  code
+  description
+  type
+  has_property
+  has_child
+}
+
+type Property {
+  type
+  value
+}
+
+code: string @index(exact, fulltext).
+description: string @index(exact, fulltext) .
+type: string @index(exact) .
```

Comment:
Not sure if indexes are premature at this stage, but dealing with small dataset so indexing tradeoffs aren't too much of a concern anyway!

![shrug](https://user-images.githubusercontent.com/43223637/85935110-dd4bcf80-b940-11ea-88bc-f0ad52c9e2eb.gif)